style/scss/_details.scss

```diff
@@ -0,0 +1,476 @@
+.main {
+  box-shadow: 0px 1px 18px 0px rgba($dark-gray, 0.35);
+  margin: 0 20px;
+  border-radius: 30px;
+  padding-bottom: 30px;
+}
+
+.back__wrapper {
+  margin: 140px 20px 20px 20px;
+}
+
+.backBtn {
+  padding-top: 15px;
+  display: flex;
+  align-items: center;
+
+  .backTxt {
+    font-weight: 500;
+    line-height: 24px;
+    font-size: 1.6rem;
+    color: $dark-gray;
+  }
+}
+
+.jd__container {
+  gap: 54px;
+  display: flex;
+  flex-direction: column;
+  margin: 0 20px;
+}
+
+.jd__header {
+  display: flex;
+  flex-direction: column;
+  gap: 54px;
+  font-size: 1.6rem;
+}
+
+.jd__main {
+  border: 1px solid rgba($black, 0.2);
+  border-radius: 16px;
+  padding: 24px 28px;
+  gap: 20px;
+  display: flex;
+  flex-direction: column;
+
+  .title {
+    font-size: 36px;
+    font-weight: 700;
+    line-height: 47px;
+    color: $green-mint;
+  }
+
+  .title__details {
+    display: flex;
+    flex-direction: column;
+    gap: 10px;
+    margin-top: 10px;
+
+    .employer {
+      color: $dark-gray;
+    }
+
+    .departament {
+      font-weight: 600;
+      font-size: 18px;
+      line-height: 27px;
+      color: $black;
+    }
+  }
+
+  .subtitles {
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    justify-content: space-between;
+
+    .info {
+      display: flex;
+      flex-direction: column;
+      gap: 10px;
+
+      .item {
+        display: flex;
+        align-items: center;
+        gap: 6px;
+        color: $dark-gray;
+        .img {
+          stroke: $dark-gray;
+          fill: $dark-gray;
+        }
+      }
+    }
+
+    .applyBtn {
+      background-color: $green-mint;
+      border: none;
+      border-radius: 8px;
+      padding: 12px 32px;
+      color: $white;
+      font-size: 18px;
+      font-weight: 500;
+      width: 100%;
+      height: 50px;

```

Comment:
Please remove the height, so that the button will be the same with the one in the navbar.